docs/openapi.yaml

```diff
@@ -713,6 +713,46 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Problem"
+  /publication/{publicationId+}/report-status:
+    get:
+      description: Get report status by publication id
+      summary: Get report status by publication id
+      operationId: getReportStatusByPublicationId
+      parameters:
+        - in: path
+          name: publicationId+
```

Comment:
Should this `+` be here?

Reply:
It's in case the URL-encoded `publicationId` includes slashes